pom.xml

```diff
@@ -229,6 +230,11 @@
                         <skip>true</skip>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>io.fabric8</groupId>
```

Comment:
It clears a warning from maven
```
[WARNING] Some problems were encountered while building the effective model for se.fortnox:rocket-fuel-impl:jar:0.1.22-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for io.fabric8:docker-maven-plugin is missing. @ line 126, column 21
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for se.fortnox:rocket-fuel-ui:jar:0.1.22-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for io.fabric8:docker-maven-plugin is missing. @ line 51, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```